index.html

```diff
@@ -149,434 +148,175 @@ <h3 id="examples">More examples</h3>
     </li>
 
-    <li class="custom-image" id="custom-image-example">
+    <li>
       <div class="ui">
         <p>A message with a custom image and CSS animation disabled</p>
-        <button aria-label="Try me! Example: A message with a custom image and CSS animation disabled">Try me!</button>
+        <button aria-label="Try me! Example: A message with a custom image and CSS animation disabled" onclick="executeExample('customImage')">
+          Try me!
+        </button>
       </div>
-      <pre class="code-sample">
-<code>Swal.fire({
-  title: 'Sweet!',
-  text: 'Modal with a custom image.',
-  imageUrl: 'https://unsplash.it/400/200',
-  imageWidth: 400,
-  imageHeight: 200,
-  imageAlt: 'Custom image',
-  animation: false
-})</code></pre>
+      <pre data-example-id="customImage"></pre>
     </li>
 
-    <li class="custom-width-padding-background" id="custom-width-padding-background">
+    <li>
       <div class="ui">
         <p>A message with custom width, padding, background and animated Nyan Cat</p>
-        <button aria-label="Try me! Example: A message with custom width, padding and background">Try me!</button>
+        <button aria-label="Try me! Example: A message with custom width, padding and background" onclick="executeExample('customWidth')">
+          Try me!
+        </button>
       </div>
-      <pre class="code-sample">
-<code>Swal.fire({
-  title: 'Custom width, padding, background.',
-  width: 600,
-  padding: '3em',
-  background: '#fff url(/images/trees.png)',
-  backdrop: `
-    rgba(0,0,123,0.4)
-    url("/images/nyan-cat.gif")
-    center left
-    no-repeat
-  `
-})</code></pre>
+      <pre data-example-id="customWidth"></pre>
     </li>
 
-    <li class="timer" id="timer-example">
+    <li>
       <div class="ui">
         <p>A message with auto close timer</p>
-        <button aria-label="Try me! Example: A message with auto close timer">Try me!</button>
+        <button aria-label="Try me! Example: A message with auto close timer" onclick="executeExample('timer')">
+          Try me!
+        </button>
       </div>
-      <pre class="code-sample">
-<code>let timerInterval
-Swal.fire({
-  title: 'Auto close alert!',
-  html: 'I will close in &lt;strong>&lt;/strong> seconds.',
-  timer: 2000,
-  onBeforeOpen: () => {
-    Swal.showLoading()
-    timerInterval = setInterval(() => {
-      Swal.getContent().querySelector('strong')
-        .textContent = Swal.getTimerLeft()
-    }, 100)
-  },
-  onClose: () => {
-    clearInterval(timerInterval)
-  }
-}).then((result) => {
-  if (
-    // <a href="#handling-dismissals" tabindex="-1">Read more about handling dismissals</a>
-    result.dismiss === Swal.DismissReason.timer
-  ) {
-    console.log('I was closed by the timer')
-  }
-})</code></pre>
+      <pre data-example-id="timer"></pre>
     </li>
 
-    <li class="rtl" id="rtl">
```

Comment:
ids are used as anchors, anchors are handy when it's needed to share a link to a particular example (e.g. https://github.com/sweetalert2/sweetalert2.github.io/blob/master/index.html#L1128)

What's the reson for dropping them in this PR?

Reply:
@zenflow FYI I reverted ids, other than that all is well :rocket: